docs/api-docs/partner/Dev-portal-overview.md

```diff
@@ -0,0 +1,100 @@
+# Developer Portal Overview
+
+<div class="otp" id="no-index">
+
+### On this page
+- [About BigCommerce apps](#about-bigcommerce-apps)
+- [Use cases](#use-cases)
+- [Developing your app](#develop-your-app)
+- [Resources](#resources)
+</div> 
+
+Using BigCommerce’s powerful APIs, SDKs, and toolkits, developers can register and build new apps on the developer portal that are installable on BigCommerce stores. Once approved, you can sell apps (or made available free of charge) to all BigCommerce merchants on the [Apps Marketplace](https://www.bigcommerce.com/apps/). You will need a [DevTools](https://devtools.bigcommerce.com/my/apps) account to register the draft app and a BigCommerce store to test the installation.
+
+## About BigCommerce apps
```

Comment:
This section appears to be a copy of the information provided in Introduction to Building Apps. Instead of including sections of that article, I think we should link to it instead.
```suggestion
## About BigCommerce apps
```